Sources/HTTPEndpoints.swift

```diff
@@ -76,7 +76,7 @@ private class LXPersistedCache {
     /// Add data to the cache; the data can be retrieved for upload later.
     func addData(data: NSData) {
         dispatch_async(self.lock, {
-            let id = ++self.currentMaxID
+			let id: UInt = UInt(self.currentMaxID += 1)
```

Comment:
@JackoPlane 
I updated this while merging your PR, but I wanted to point out that your update in this instance was not quite right.

`UInt(self.currentMaxID += 1)` always evaluates to `0`, because the expression `self.currentMaxID += 1` does not return anything -- or more accurately, returns `()` -- and `UInt(())` evaluates to `0`. Thus `id` would always be `0`, even while `currentMaxID` is being incremented, which is not what we want!

Anyway, I updated it while merging to correct the behavior. Otherwise, thanks for the PR!